services/web/src/lib/reusable-components/test-form.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+import { Button } from '@/components/ui/button';
+import { Form } from '@/components/ui/form';
+import { DropdownComponent } from '@/lib/reusable-components/dropdown';
+
+const FormSchema = z.object({
+    fruit: z.string({
+        required_error: 'Please select a fruit',
+    }),
+});
+
+export function TestForm() {
+    const form = useForm<z.infer<typeof FormSchema>>({
+        resolver: zodResolver(FormSchema),
+    });
+
+    const label: string = 'Fruits';
```

Comment:
There is no need to create variables for these strings you can directly put them where you call your component.